man/data.table.Rd

```diff
@@ -403,6 +403,15 @@ print(DT["b", v2:=84L, on="x"])       # subassign to new column by reference (NA
 
 DT[, m:=mean(v), by=x][]              # add new column by reference by group
                                       # NB: postfix [] is shortcut to print()
+
+# Injecting function into env parameter
+DT <- data.table(a = rep(1:2, each = 5), b = 1:10)
+
+DT[, f(b), by = a, env = list(f = "sum")]  # Use function name as a string
+DT[, f(b), by = a, env = list(f = "sum"), verbose = TRUE]  # Preview function name injection
+DT[, f(b), by = a, env = list(f = sum)]  # Use function object directly
+DT[, f(b), by = a, env = list(f = sum), verbose = TRUE]  # Preview function object injection
```

Comment:
what do you think about replacing these four separate examples with this one?
```r
> DT[, .(f=f(b), gval=g(b)), by = a, env = list(f = "sum", g=mean), verbose=TRUE]
Argument 'by' after substitute: a
Argument 'j' after substitute: .(sum = sum(b), gval = (function (x, ...)  UseMethod("mean"))(b))
Detected that j uses these columns: [b]
Finding groups using forderv ... forderReuseSorting: opt not possible: is.data.table(DT)=0, sortGroups=0, all1(ascArg)=1
forder.c received 10 rows and 1 columns
forderReuseSorting: opt=0, took 0.000s
0.000s elapsed (0.000s cpu) 
Finding group sizes from the positions (can be avoided to save RAM) ... 0.000s elapsed (0.000s cpu) 
lapply optimization is on, j unchanged as 'list(sum(b), (function (x, ...) '
GForce is on, but not activated for this query; left j unchanged (see ?GForce)
Old mean optimization is on, left j unchanged.
Making each group and running j (GForce FALSE) ... 
  memcpy contiguous groups took 0.000s for 2 groups
  eval(j) took 0.000s for 2 calls
0.000s elapsed (0.000s cpu) 
       a   sum  gval
   <int> <int> <num>
1:     1    15     3
2:     2    40     8
```
it shows that you can do either quoted or not
and I don't think it is necessary to do both verbose=TRUE and FALSE. please keep only verbose=TRUE

Reply:
sure 👍

Reply:
done